lib/experimental/Navigation/Header/Breadcrumbs/index.tsx

```diff
@@ -20,10 +19,52 @@ import { cn, focusRing } from "@/lib/utils"
 import { NavigationItem } from "../../utils"
 
 import { IconType } from "@/components/Utilities/Icon"
-import { useEffect, useRef, useState } from "react"
+import { Fragment, useEffect, useRef, useState } from "react"
 
-export type BreadcrumbItemType = NavigationItem & {
-  icon?: IconType
+export type BreadcrumbItemType =
+  | (NavigationItem & {
+      icon?: IconType
+    })
+  | {
+      loading: true
+    }
+
+function BreadcrumbSkeleton() {
+  return (
+    <div
+      className="h-4 w-24 animate-pulse rounded-xs bg-f1-background-secondary"
+      aria-label="Loading breadcrumb"
```

Comment:
Dammit, forgot about this hardcoded copy. These are the kinds of things I feel should be provided with a context @nlopin .